components/aws/common/common-cloudwatch-logs.mjs

```diff
@@ -0,0 +1,118 @@
+import aws from "../aws.app.mjs";
```

Comment:
Optional:

I think is not necessary to have a filename like `common-cloudwatch-logs.mjs` especially if it is inside the `common` dir. My suggestion is to rename these files with just `cloudwatch-logs.mjs` as an example

Reply:
Yeah, that sounds good. I'll create another PR to rename all common files after everything is approved and merged!